docs/issues.rst

```diff
@@ -0,0 +1,12 @@
+****************
+Reporting Issues
+****************
+
+TARDIS is under constant development. Thus, changes are high that some things
```

Comment:
I think you meant "chances are high", but since it is under constant development, even the "changes are high". What should exactly go here ? I think "chances" :grin: